From 48723f28aa4451b1793373c35527d3627b001b7e Mon Sep 17 00:00:00 2001
From: Daniel Molkentin <dmolkentin@suse.com>
Date: Thu, 1 Mar 2018 10:24:29 +0100
Subject: [PATCH] 90kernel-modules: Ensure PCI host modules are included

The includes modules like the Intel Volume Management Device driver,
which is required to boot from disk on some systems.

Reference: boo#1079924
---
 modules.d/90kernel-modules/module-setup.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/90kernel-modules/module-setup.sh b/modules.d/90kernel-modules/module-setup.sh
index 076f1e8a..94592e6b 100755
--- a/modules.d/90kernel-modules/module-setup.sh
+++ b/modules.d/90kernel-modules/module-setup.sh
@@ -17,6 +17,7 @@ installkernel() {
             "=drivers/input/serio" \
             "=drivers/input/keyboard" \
             "=drivers/usb/storage" \
+            "=drivers/pci/host" \
             ${NULL}
 
         instmods \

